airbyte-server/src/main/java/io/airbyte/server/config/WorkspaceApiFactory.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.config;
+
+import io.airbyte.analytics.TrackingClient;
+import io.airbyte.analytics.TrackingClientSingleton;
+import io.airbyte.commons.temporal.TemporalClient;
+import io.airbyte.config.EnvConfigs;
+import io.airbyte.config.persistence.ConfigRepository;
+import io.airbyte.config.persistence.SecretsRepositoryReader;
+import io.airbyte.config.persistence.SecretsRepositoryWriter;
+import io.airbyte.config.persistence.StreamResetPersistence;
+import io.airbyte.config.persistence.split_secrets.SecretPersistence;
+import io.airbyte.db.Database;
+import io.airbyte.persistence.job.JobPersistence;
+import io.airbyte.persistence.job.WorkspaceHelper;
+import io.airbyte.server.handlers.ConnectionsHandler;
+import io.airbyte.server.scheduler.EventRunner;
+import io.airbyte.server.scheduler.TemporalEventRunner;
+import io.airbyte.validation.json.JsonSchemaValidator;
+import io.airbyte.workers.helper.ConnectionHelper;
+import io.micronaut.context.annotation.Factory;
+import jakarta.inject.Named;
+import jakarta.inject.Singleton;
+import org.jooq.DSLContext;
+
+@Factory
+public class WorkspaceApiFactory {
+
+  @Singleton
+  EnvConfigs envConfigs() {
```

Comment:
we shouldn't use the `EnvConfig` but the micronaut application values like in https://github.com/airbytehq/airbyte/pull/20211/files#diff-ac6719193135419f98d16e4e7ae96685864ee34a7302ac7ce428dc142736b17aR40

Reply:
I think I already handled this, double-checking...

Reply:
...and only partially handled.

Reply:
Sorry...I missed this in my review.  We should definitely avoid using `EnvConfig` and convert any code that does use it to get the configuration via injection.